bridge/functions.php

```diff
@@ -10,12 +10,38 @@ function dd_tracing_enabled()
         return true;
     }
 
-    $value = trim(strtolower($value));
-    if ($value === '0' || $value === 'false') {
-        return false;
-    } else {
+    $value = strtolower(trim($value));
+    return !($value === '0' || $value === 'false');
+}
+
+/**
+ * Checks the `DD_DISABLE_URI` env var for routes to disable the tracer on
+ *
+ * @param string|null $requestUri
+ * @return bool
+ */
+function dd_tracing_route_enabled($requestUri = null)
+{
+    $value = getenv('DD_DISABLE_URI');
+    if (false === $value) {
+        // Not setting the env means we default to enabled.
+        return true;
+    }
+    if (null === $requestUri) {
+        $requestUri = isset($_SERVER['REQUEST_URI']) ? $_SERVER['REQUEST_URI'] : null;
+    }
+    if (!$requestUri) {
         return true;
     }
+
+    $uris = explode(',', $value);
+    foreach ($uris as $uri) {
+        $uriRegex = str_replace('\*', '.*', preg_quote(trim($uri), '~'));
+        if (1 === preg_match('~^' . $uriRegex . '$~', $requestUri)) {
```

Comment:
Yeah, it doesn't really support scenarios where you have a lot of different query params that can  be applied in any order. I can remove the query param test from the tests to show we don't support it officially. :)

Reply:
Do you think then that it would be a good idea to remove the query string parameter alltogether? something like using as the value the result of `strtok($_SERVER['REQUEST_URI'], '?')`. We can always add it back after talking to the various teams.

Reply:
I love that idea! Done. :)